calc > try{echo"hola";} catch(CustomException $e){echo"hola";}

calc > $e="";

calc > try{echo"hola";} catch(CustomException $e){echo"hola";}

